src/client/cockpit-client

```diff
@@ -295,24 +294,16 @@ class ExternalCockpitWs:
 
 class InternalCockpitWs:
     def start(self):
-        self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        # Static port is needed for WebKit local storage to persist across runs
-        self.socket.bind(('127.0.90.90', 9090))
-        self.socket.listen()
-
         self.write_config()
-        self.ws = subprocess.Popen([f'{libexecdir}/cockpit-ws'],
+        host = '127.0.0.90'
+        self.ws = subprocess.Popen([f'{libexecdir}/cockpit-ws', f'--address={host}'],
```

Comment:
I wonder if you'd also pass `--port={port}` and return the same variable.  The fact that 9090 is the default port is now encoded in two separate places (in -ws itself, plus the assumption here), and as we know, we never actually test this one ;)

Reply:
Sure! Done.